### PR TITLE
Ts 673

### PR DESCRIPTION
**Pull request by tomskikh**

### CODE REVIEW

File: src/test/scala/com/bwsw/tstreams/benchmark/BenchmarkConfig.scala [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+    short = 'p',
+    descr = "path to master node in ZooKeeper")
+
+  val stream: ScallopOption[String] = opt[String](
+    default = Some(Default.stream),
+    descr = s"stream name (${Default.stream} by default)")
+
+  val partitions: ScallopOption[Int] = opt[Int](
+    default = Some(Default.partitions),
+    descr = s"amount of partitions on stream (${Default.partitions} by default)")
```
Comment (placccebo) [inline src/test/scala/com/bwsw/tstreams/benchmark/BenchmarkConfig.scala]: amount of  stream partitions

File: src/test/scala/com/bwsw/tstreams/benchmark/BenchmarkConfig.scala [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+
+  val iterations: ScallopOption[Int] = opt[Int](
+    default = Some(Default.iterations),
+    descr = s"amount of measurements (${Default.iterations} by default)")
+
+  val dataSize: ScallopOption[Int] = opt[Int](
+    default = Some(Default.dataSize),
+    descr = s"size of data sent in each transaction (${Default.dataSize} by default)")
```
Comment (placccebo) [inline src/test/scala/com/bwsw/tstreams/benchmark/BenchmarkConfig.scala]: in -> to

File: src/test/scala/com/bwsw/tstreams/benchmark/BenchmarkConfig.scala (hunk 1 above)
Comment (tomskikh) [reply src/test/scala/com/bwsw/tstreams/benchmark/BenchmarkConfig.scala]: fixed

File: src/test/scala/com/bwsw/tstreams/benchmark/BenchmarkConfig.scala (hunk 2 above)
Comment (tomskikh) [reply src/test/scala/com/bwsw/tstreams/benchmark/BenchmarkConfig.scala]: fixed

**Pull request merged**